examples/react/start-basic/package.json

```diff
@@ -12,6 +12,7 @@
     "@tanstack/react-router": "^1.120.5",
     "@tanstack/react-router-devtools": "^1.120.6",
     "@tanstack/react-start": "^1.120.5",
+    "@tanstack/router-core": "^1.120.5",
```

Comment:
this should not be necessary as it is a dep of `@tanstack/react-router`

Reply:
You are right that, this is not needed in default `npm` setup, but it is necessary in the setup our company and others like it could be using: 

`@tanstack/react-router-devtools` requires `@tanstack/router-core` as a **peer** and if you use a stricter dependency manager like `pnpm` it means that the implicit dependencies of `@tanstack/react-router` are **not shared** with the rest of your dependencies, a.k.a. `@tanstack/react-router-devtools` doesn't have access to the dependencies of `@tanstack/router-core`.

This is meaningful and intentional for portability, so that developers need to declare what dependencies they use, and not be able to import some implicit dependency of a dependency.

Does it make sense? I can try to rephrase if i wasn't able to express very well.